custom-post-type-ui.php

```diff
@@ -672,11 +673,6 @@ function cptui_register_single_taxonomy( $taxonomy = [] ) {
 		$rest_base = $taxonomy['rest_base'];
 	}
 
-	$rest_controller_class = null;
-	if ( ! empty( $post_type['rest_controller_class'] ) ) {
-		$rest_controller_class = $post_type['rest_controller_class'];
-	}
-
```

Comment:
Please remove this change portion from this branch, as it's part of a separate issue opened that needs to be amended instead of removed :)

Reply:
I removed.